library/libraries.js

```diff
@@ -1376,6 +1376,17 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             return { version };
         }
     },
+    'Wix': {
+        icon: 'wix',
+        url: 'https://www.wix.com/',
+        npm: null,
+        test: function (win) {
+            if (win.wixBiSession) {
```

Comment:
Is there a secondary check that we could include here with `&&`?

Reply:
We got this check from the Wix engineers themselves, so I suspect this is sufficient.